wp-modules/editor/js/src/utils/validateInput.ts

```diff
@@ -0,0 +1,15 @@
+const defaultPattern = new RegExp( /([^a-z0-9 -]+)/gi );
```

Comment:
`defaultPattern` (and its usage as a default param) might be unneeded, but I like how simple it is to test.